### PR TITLE
Bump diplomat

### PR DESCRIPTION
**Pull request by robertbastian**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -799,7 +799,6 @@ dependencies = [
 [[package]]
```
Comment (sffc) [inline Cargo.lock:799]: Are the diffs to Cargo.lock expected?

Comment (Manishearth) [reply Cargo.lock:799]: yeah i'm surprised that happened.

**Review by Manishearth — APPROVED** [review]
r=me but you'll need to fix gn

**Review by sffc — DISMISSED** [review]

**Comment by robertbastian** [comment]
https://github.com/unicode-org/icu4x/issues/3006

**Review by sffc — APPROVED** [review]

**Review by Manishearth — APPROVED** [review]

**Pull request merged**